### ISSUE TITLE
Can't quit Feishin

### ISSUE THREAD

**Issue by toasty-sunbeam** [jeffvli/feishin#350]
I love this app so much I can't quit it! Seriously, why can't I quit?

When I quit Feishin, either by right-clicking on its icon in the dock or by choosing Quit from the menu, the window closes, but the app remains open. The dot remains under its icon on the dock, and it continues playing music if music was playing when it was closed. (Possibly related to #4) Only using MacOS' "Force Quit" command makes the dot go away. The app also can't be reopened until it's force-quit and restarted.

## Expected Behavior

Feishin should quit when users use the Quit command, and music should stop playing.

## Current Behavior

The window closes, but the dot doesn't disappear from under its dock icon, and music continues playing.

## Your Environment

-   Application version (e.g. v0.1.0) : 0.5.0
-   Operating System and version (e.g. Windows 10) : MacOS 14.0 (Intel chip)
-   Server and version (e.g. Navidrome v0.48.0) : Navidrome 0.49.3
-   Node version (if developing locally) : n/a

**Comment by kgarner7** [comment]
I am unable to still reproduce this issue; command + Q and Icon > quit are both closing for me. Is this issue still present?

**Comment by kgarner7** [comment]
Since this issue is not seen on current development build, I am going to mark it resolved. If this is still seen, please feel free to reopen it!

**Comment by jamie6king** [comment]
Currently having this issue, with v0.9.0 & MacOS 14. Command-Q & Feishin > Quit & right-click > Quit only hides the application.

**Comment by toasty-sunbeam** [comment]
I think I was able to resolve this issue by toggling Settings > Window > Show Tray. It was acting as if "Exit to tray" was turned on, even though it wasn't. I turned "Show Tray" off and on and now the app exits normally.

It might be worth taking another look at how that option works on Mac. I assume in this context "tray" refers to the list of icons next to the clock in the menu bar at the top of the screen? Feishin doesn't show up in that list regardless of how "Show tray" is set. Current behavior is confusing IMO. Anyway, I can exit Feishin normally again, so I'm happy.
